lib/core/base/virtual-node/virtual-node.js

```diff
@@ -49,14 +49,15 @@ class VirtualNode extends AbstractVirtualNode {
 	// abstract Node properties so we can run axe in DOM-less environments.
 	// add to the prototype so memory is shared across all virtual nodes
 	get props() {
-		const { nodeType, nodeName, id, multiple } = this.actualNode;
+		const { nodeType, nodeName, id, multiple, textContent } = this.actualNode;
```

Comment:
I suspect this is going to be faster, as it won't require the browser to compute the text content of every node in the page.

```suggestion
		const { nodeType, nodeName, id, multiple, nodeValue } = this.actualNode;
```